physical/consul/consul.go

```diff
@@ -224,13 +229,47 @@ func SetupSecureTLS(ctx context.Context, consulConf *api.Config, conf map[string
 	return nil
 }
 
+// ExpandedCapabilitiesAvailable tests to see if Consul has KVGetOrEmpty and 128 entries per transaction available
+func (c *ConsulBackend) ExpandedCapabilitiesAvailable(ctx context.Context) bool {
+	available := false
+
+	maxEntries := 128
+	ops := make([]*api.TxnOp, maxEntries)
+	for i := 0; i < maxEntries; i++ {
+		ops[i] = &api.TxnOp{KV: &api.KVTxnOp{
+			Key:  c.path + "some-path-that-hopefully-does-not-exist-abc123",
+			Verb: api.KVGetOrEmpty,
```

Comment:
I don't see the go.mod change, which I guess is why we're getting the check failure here?

Reply:
Ah, apparently my git alias for moving ENT changes to OSS isn't smart enough to see go.mod/go.sum changes. I'll fix this.

Reply:
The patch wouldn't apply cleanly from my ENT PR (possibly because those files are conflicting there) so I just ran the same process manually here that I did there - upgrade consul/api, ran `go mod tidy`.